frameworks/OCaml/opium/opium-haproxy.dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ocaml/opam2:alpine
```

Comment:
I asked about this image here: https://discuss.ocaml.org/t/psa-ocaml-opam2-docker-images-updated-to-4-08-0-release/3948/14?u=rbjorklin and you might want to try `ocurrent/opam:alpine-3.12-ocaml-4.11` if you want to use an alpine image with `4.11`. See: https://hub.docker.com/r/ocurrent/opam/tags?page=1&name=ocaml-4.11

Reply:
nice, I've been searching for one in the opam2 hub, but didn't find anything, generally I've had many troubles finding good docs

Reply:
replaced it and locally it builds without any issues

unrelated to this, but I wanted to test out opam's lock files, but the build didn't work when using a lock file